week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
+    /*왜 100%로 하면 스크롤바가 생기는지?*/
+    background-color: #c1bcea;
+}
+.top_end{
+    height: 10%;
+    /*border: 2px solid blue;*/
+}
+.background{
+    display: flex;
+    flex : 1;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    /* border: 2px solid black; */
+    height: 80%;
+    justify-content: space-around;
+    
+}
+
+.top{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    /* border: 2px solid red; */
+    height: 20%;
+    width: 30%;
+    
+    
+}
+
+.score{
+    color: white;
+    width: 40%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    /* border: 2px solid green; */
+    font-size: 45px;
+
+}
```

1. 텍스트는 width를 설정할 필요가 없습니다. 텍스트의 길이와 font-size에 의해 자동으로 크기가 설정됩니다.
2. 불필요한 자식 요소를 삭제하였으므로, 자식 요소에 대한 불필요한 코드 또한 삭제하는 것이 좋습니다.
```suggestion
body .result .score {
    font-size: 45px;
}
```